authors/apps/authentication/serializers.py

```diff
@@ -34,6 +34,7 @@ class LoginSerializer(serializers.Serializer):
     email = serializers.CharField(max_length=255)
     username = serializers.CharField(max_length=255, read_only=True)
     password = serializers.CharField(max_length=128, write_only=True)
+    
```

Comment:
Why a blank line addition here?

Reply:
Remove unnecessary white-spaces

Reply:
Please remove your edit on this file. Your task doesn't require you to edit this file?